cfgov/paying_for_college/models/disclosures.py

```diff
@@ -214,6 +226,8 @@ class School(models.Model):
     url = models.TextField(blank=True)
     degrees_predominant = models.TextField(blank=True)
     degrees_highest = models.TextField(blank=True)
+    program_count = models.IntegerField(blank=True, null=True)
+    program_most_popular = JSONField(blank=True, null=True)
```

Comment:
Maybe the cow is out of the barn at this point, but is it worth doing some kind of check for Postgres on tests that touch this behavior, a la [this code](https://github.com/cfpb/cfgov-refresh/blob/e0663d69a87d4af239edecf6c91d5e8fdca52394/cfgov/prepaid_agreements/tests/test_views.py#L41) for the prepaid agreements use of Postgres search?

Reply:
I would vote not to support testing with other databases, but I'm unreasonably opinionated on this.

Reply:
Now that this has been merged, the cfgov-refresh tests fail if you don't have a `TEST_DATABASE_URL` environment variable set to point to a Postgres DB. The [current default](https://github.com/cfpb/cfgov-refresh/blob/10a0d58b666ee95c721dd37ff2f78b9bafa26249/cfgov/cfgov/settings/test.py#L7) is to try to use SQLite if this isn't defined, which causes several JSONField-related tests to fail with errors like "InterfaceError: Error binding parameter 14 - probably unsupported type." (To replicate, try running tox with that environment variable unset.) Travis [explicitly uses Postgres](https://github.com/cfpb/cfgov-refresh/blob/10a0d58b666ee95c721dd37ff2f78b9bafa26249/.travis.yml#L42) when running the tests, so this problem doesn't show up there.

We should either change the settings file and the docs ([1](http://cfpb.github.io/cfgov-refresh/installation/#install-postgres), [2](http://cfpb.github.io/cfgov-refresh/python-unit-tests/#running-tests)) to remove any reference of supporting SQLite, and require Postgres; OR mark the relevant tests with [`skipUnless`](https://github.com/cfpb/cfgov-refresh/blob/e0663d69a87d4af239edecf6c91d5e8fdca52394/cfgov/prepaid_agreements/tests/test_views.py#L56). I probably have a weak preference for the latter, but don't feel very strongly either.

Reply:
I'll mark relevant tests. Thanks for the follow-up.